REPORT.md

```diff
@@ -0,0 +1,76 @@
+# Reporting capabilities within zAppBuild
+
+zAppBuild provides a set of reporting capabilities, which are part of the build framework itself to address some common demands of mainframe development teams.
+
+Developers are particularly interested if
+* a change of an element impacts other application components which are managed in a different repository.
+* their changes on an isolated feature branch potentially cause a conflict with some concurrent development done by others within the same repository.
+
+While during the analysis phase of a new task, the developer leverages application dependency understanding tools such as IBM Application Discovery, it is beneficial to enable also the build framework to automatically generate reports to answer the above questions.
```

Comment:
Suggestion: During the analysis phase, while a developer can leverages application dependency tools such as IBM Application Discovery, it is also beneficial to enable the build framework to automatically generate reports to answer the above questions.